docs/src/getting_started.md

```diff
@@ -68,28 +68,28 @@ docs $
 
 ### Prerequisites
 
-DocumenterVitepress.jl is completely self-contained and installs all of its dependencies (including its own isolated version of `npm`) automatically. 
+DocumenterVitepress.jl is completely self-contained and installs all of its dependencies (including its own isolated version of `npm`) automatically.
 
-However, to view your documentation live when developing locally, you will need to install `npm` and instantiate the 
+However, to view your documentation live when developing locally, you will need to install `npm` and instantiate the
 
 VitePress can be used on its own, or be installed into an existing project. In both cases, you can install it with:
 
 ::: code-group
 
 ```sh [npm]
-npm add -D vitepress
+$ npm add -D vitepress
```

Comment:
I don't know how to feel about the dollar sign here. The idea is to just copy paste, having that symbol will cause some trouble to newcomers.